### PR TITLE
Add: SDL_ShowCursor

### PR DESCRIPTION
**Pull request by kussakaa**

### CODE REVIEW

File: libs/zsdl/src/sdl2.zig [hunk 1]
```diff
@@ -1450,6 +1450,17 @@ pub const GameController = opaque {
     extern fn SDL_GameControllerGetButton(controller: *GameController, button: c_int) u8;
 };
 
+//--------------------------------------------------------------------------------------------------
+//
+// Cursor
+//
+//--------------------------------------------------------------------------------------------------
+
```
Comment (hazeycode) [inline libs/zsdl/src/sdl2.zig:1458]: These section comments correspond to the categories defined by the SDL docs (see https://wiki.libsdl.org/SDL2/APIByCategory). Please move `showCursor` under the "Mouse Support" section comment and delete this one

File: libs/zsdl/src/sdl3.zig [hunk 2]
```diff
@@ -1481,6 +1481,22 @@ pub const Gamepad = opaque {
     extern fn SDL_GetGamepadButton(controller: *Gamepad, button: c_int) u8;
 };
 
+//--------------------------------------------------------------------------------------------------
+//
+// Cursor
+//
+//--------------------------------------------------------------------------------------------------
```
Comment (hazeycode) [inline libs/zsdl/src/sdl3.zig:1488]: Same here

**Review by hazeycode — CHANGES_REQUESTED** [review]

**Comment by kussakaa** [comment]
I didnt know about the locations of functions, now I'll know. Thank you!

**Review by hazeycode — APPROVED** [review]

**Comment by hazeycode** [comment]
Thanks!

**Pull request merged**